karavan-app/src/main/docker/Dockerfile

```diff
@@ -26,14 +26,16 @@ RUN mkdir /opt/app
 COPY target/*-runner.jar /opt/app/karavan.jar
 ENV JAVA_OPTS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager -XX:-UseG1GC -XX:+UseZGC"
 EXPOSE 8080
-CMD ["java", \
-    "--add-modules", "java.se", \
-    "--add-exports", "java.base/jdk.internal.ref=ALL-UNNAMED", \
-    "--add-opens", "java.base/java.lang=ALL-UNNAMED", \
-    "--add-opens", "java.base/java.nio=ALL-UNNAMED", \
-    "--add-opens", "java.base/sun.nio.ch=ALL-UNNAMED", \
-    "--add-opens", "java.management/sun.management=ALL-UNNAMED", \
-    "--add-opens", "jdk.management/com.ibm.lang.management.internal=ALL-UNNAMED", \
-    "--add-opens", "jdk.management/com.sun.management.internal=ALL-UNNAMED", \
-    "-jar", "/opt/app/karavan.jar"]
+
+CMD exec "java" \
+    ${JAVA_OPTS} \
+    "--add-modules" "java.se", \
```

Comment:
These options was needed when Karavan used Hazelcast for caching. 
Hazelcast was replaced by plain Maps in 4.7. These options should be removed.

Reply:
> These options was needed when Karavan used Hazelcast for caching. Hazelcast was replaced by plain Maps in 4.7. These options should be removed.

Thanks for the reply @mgubaidullin 

So... I removed the default values not needed anymore. But kept the JAVA_OPTS on the command line because that's nice to have for so many reasons, like setup xms/xmx, or others like -Xrunjdwp for remote debugging (that's how I discovered that my git repo did not had a default branch created, and the logs were not giving any hint)

Reply:
@thgruiz Agree. Keep `JAVA_OPTS`

Reply:
Should all those "--add-opens" etc not be removed if it was a hazelcast workaround that is no longer needed.

Reply:
yes. All these
```
    "--add-modules", "java.se", \
    "--add-exports", "java.base/jdk.internal.ref=ALL-UNNAMED", \
    "--add-opens", "java.base/java.lang=ALL-UNNAMED", \
    "--add-opens", "java.base/java.nio=ALL-UNNAMED", \
    "--add-opens", "java.base/sun.nio.ch=ALL-UNNAMED", \
    "--add-opens", "java.management/sun.management=ALL-UNNAMED", \
    "--add-opens", "jdk.management/com.ibm.lang.management.internal=ALL-UNNAMED", \
    "--add-opens", "jdk.management/com.sun.management.internal=ALL-UNNAMED", 
```
changes are related to Hazelcast and need to be removed

Reply:
@thgruiz are you able to remove all of these `--add-opens` in the script files, thanks.

Reply:
Sorry for the delay

@davsclaus Done!